spec/support/factory_girl.rb

```diff
@@ -1,4 +1,5 @@
 require 'factory_girl'
+require 'factory_girl_rails'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.